tests/raftstore/cluster.rs

```diff
@@ -110,11 +110,14 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn start_with_strategy(&mut self, strategy: Vec<Strategy>) {
-        let mut sim = self.sim.wl();
         for engine in &self.dbs {
-            let node_id = sim.run_node(0, self.cfg.clone(), engine.clone(), strategy.clone());
```

Comment:
no big different, why I want to move inner is to avoid using `drop sim` before sleep.